CMakeLists.txt

```diff
@@ -231,7 +231,7 @@ target_compile_options(
 )
 
 # add additional compile warning flags depending on the compiler
-ADJUST_COMPILER_WARNINGS()
+ADJUST_COMPILER_WARNINGS2()
```

Comment:
This is intentional, but will not be merged as this.
Verifying some hypotheses that we have this function shadowed by a similar one in umd...